src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -28,164 +28,36 @@
 
 use Cache;
 use Cart;
-use CartRule;
-use Configuration;
 use Context;
-use Country;
 use Exception;
 use Hook;
 use Link;
 use PrestaShop\PrestaShop\Adapter\Image\ImageRetriever;
 use PrestaShop\PrestaShop\Adapter\Presenter\PresenterInterface;
-use PrestaShop\PrestaShop\Adapter\Presenter\Product\ProductLazyArray;
-use PrestaShop\PrestaShop\Adapter\Presenter\Product\ProductListingLazyArray;
-use PrestaShop\PrestaShop\Adapter\Presenter\Product\ProductListingPresenter;
-use PrestaShop\PrestaShop\Adapter\Product\PriceFormatter;
-use PrestaShop\PrestaShop\Adapter\Product\ProductColorsRetriever;
 use PrestaShop\PrestaShop\Core\Product\ProductPresentationSettings;
 use Product;
-use ProductAssembler;
-use Symfony\Contracts\Translation\TranslatorInterface;
-use TaxConfiguration;
-use Tools;
 
 class CartPresenter implements PresenterInterface
 {
-    /**
-     * @var PriceFormatter
-     */
-    private $priceFormatter;
-
     /**
      * @var Link
      */
     private $link;
 
-    /**
-     * @var TranslatorInterface
-     */
-    private $translator;
-
     /**
      * @var ImageRetriever
      */
     private $imageRetriever;
 
-    /**
-     * @var TaxConfiguration
-     */
-    private $taxConfiguration;
-
     /**
      * @var ProductPresentationSettings
      */
     protected $settings;
 
-    /**
-     * @var ProductAssembler
-     */
-    protected $productAssembler;
```

Comment:
This is a BC break, please document it

Reply:
Eventually I moved back those functions into CartPresenter, because it was breaking, I also had to set them as public, because they now must be called from the lazy cart...